app/models/miq_queue.rb

```diff
@@ -641,6 +639,29 @@ def self.optional_values(options, keys = [:zone])
 
   private_class_method :optional_values
 
+  def self.messaging_client_options
+    queue_settings = Settings.prototype[queue_type]
```

Comment:
Do we want an env-based override for this also?

I think pods will get this before the appliance (planning to open a PR today), so it would be good to easily enable it there.

Reply:
:+1:

Reply:
@carbonin how about:

```ruby
def self.queue_type
  ENV["QUEUE_TYPE"] || Settings.prototype.queue_type
end
```

Reply:
Lol totally misread what was going on. Thanks